### PR TITLE
Refactor/centralize toolbar

### PR DESCRIPTION
**Pull request by marcoafsantana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,22 @@
 
 Sistema de gerenciamento de estoques, lojas e incentivo ao colaborador
 
-### Como rodar?
+### Como rodar o projeto?
 
 1. Abra a pasta `\estoquei-project` no terminal;
 2. digite `npm install` para instalar as dependências;
 3. Rode o servidor com o comando `npx json-server --watch src/assets/data/database.json`;
 4. Rode o front com o comando `npm start`.
```
Comment (sbjsouza) [inline README.md:10]: What do you think about deleting these lines? I think that they're already present at section `Como rodar os testes`

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -2,9 +2,22 @@
...
+
+
+### Como rodar os testes?
```
Comment (sbjsouza) [inline README.md:13]: I think that it's more about how to run our project. What do you think about?

```suggestion
```

**Review by sbjsouza — COMMENTED** [review]
Good work! Keep going ;)

#### But please, don't forget to check some comments that I gave you!

File: README.md (hunk 2 above)
Comment (marcoafsantana) [reply README.md:13]: In fact, they could have been just one point. I just split them to clarify that when you're going to run tests, the project should be up. It wasn't very clear to me when i first runt he project and needed to run the tests. So i will keep them separated but on ""Como rodar os testes?"" i wont repeat the same text, just reference it.

Comment (sbjsouza) [reply README.md:13]: Nice, thank you by your feedback!

Comment (sbjsouza) [reply README.md:13]: @marcoafsantana Feel free to merge your PR.

**Pull request merged**